.gitignore

```diff
@@ -3,6 +3,7 @@
 *.swo
 .bazelrc.local
 .cache
+.devcontainer/
```

Comment:
I wonder if as a future step we would want to create a new project template repo with our recommended container setup.